src/assets/scss/_recordedit.scss

```diff
@@ -1,441 +1,373 @@
 @use 'sass:map';
```

Comment:
Same comment that I made about `_input-switch.scss` applies to this. Please avoid running formatters on the whole file and only run it on the lines that you've changed. Also make sure your formatter is honoring our style guide (you can use `.prettierrc.js` or `.eslintrc.js` that I've already added to the project).